.github/workflows/js-tests.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  run_tests:
+    name: JS
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04 ]
+        node-version: [ 12 ]
+        python-version: [ 3.8 ]
+        include:
+          - os: ubuntu-20.04
+            path: ~/.cache/pip
```

Comment:
Is this `include` block still needed?  It looks like it just defines the `path` variable, but it doesn't seem to be used anywhere now that `${pip cache dir}` is used.

Reply:
You are right, I've removed it.